docs/history/changelog.md

```diff
@@ -50,7 +51,8 @@ Per the [normative and non-normative content and changes policy](https://docs.go
     * Merge Publication policy and Licensing pages into Publish page [#986](https://github.com/open-contracting/standard/pull/986) [#1012](https://github.com/open-contracting/standard/pull/1012).
     * Replace guidance on publication levels [#980](https://github.com/open-contracting/standard/pull/980) [#1013](https://github.com/open-contracting/standard/pull/1013) [#1045](https://github.com/open-contracting/standard/pull/1045).
   * Various minor improvements [#1051](https://github.com/open-contracting/standard/pull/1051) [#1080](https://github.com/open-contracting/standard/pull/1080) [#1083](https://github.com/open-contracting/standard/pull/1083) [#1085](https://github.com/open-contracting/standard/pull/1085) [#1091](https://github.com/open-contracting/standard/pull/1091) [#1130](https://github.com/open-contracting/standard/pull/1130) [#1227](https://github.com/open-contracting/standard/pull/1227) [#1299](https://github.com/open-contracting/standard/pull/1299) [#1337](https://github.com/open-contracting/standard/pull/1337) [#1384](https://github.com/open-contracting/standard/pull/1384).
-
+* Reference
+  * [#1429](https://github.com/open-contracting/standard/pull/1429) Update organization identifier reference.
```

Comment:
Probably best not to have a reference sub-heading in the iterative improvements section of the changelog. Suggest updating the entry as follows and moving it to the end of the iterative improvements changelog:

```suggestion
* [#1429](https://github.com/open-contracting/standard/pull/1429) Update organization identifier uris in examples.
```